### PR TITLE
[dcl.init.ref, over.ics.ref, over.ics.rank] Avoid saying function lvalue

### PR DESCRIPTION
**Pull request by frederick-vs-ja**
Fixes cplusplus/CWG#469.

Note that we can simplify the wording in [[over.ics.ref] p3](https://eel.is/c++draft/over.ics.ref#3) by the fact that an lvalue is of either an object type or a function type.

### CODE REVIEW

**Comment by tkoeppe** [comment]
@jensmaurer We can apply this if you think it's fine from a Core perspective.

**Review by jensmaurer — APPROVED** [review]
@tkoeppe , I think this is editorially-ok from a CWG perspective.

**Comment by tkoeppe** [comment]
OK, then off we go -- thanks!
